test/functional/repository/basic-methods/repository-basic-methods.ts

```diff
@@ -27,6 +27,8 @@ describe("repository > basic methods", () => {
     let connections: Connection[];
     before(async () => connections = await createTestingConnections({
         entities: [Post, Blog, Category, UserEntity, QuestionEntity],
+        entityFactory:  new OldEntityFactory()
```

Comment:
why do we need it now? Are tests failing without old entity factory now?

Reply:
https://github.com/typeorm/typeorm/blob/d3ee978501a4f1b74c99e2fc24ed5d8105919322/test/functional/repository/basic-methods/repository-basic-methods.ts#L165
Yes, we need it, otherwise this line is failing. With new entity  factory fields aren't initialized. We could remove that part of test if we don't want to use OldEntityFactory(if it's just temporary).